doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,84 @@ menu:
-relevant and update their result sets only if they need to, for example, when
-there are new rows or the values used in dataflow computations have changed.
-
-When you query one of your views, Materialize can simply return the dataflow's
-result set, which should always be faster than computing the answer from
-scratch.
-
-## Materialize vs. other methodologies
-
-### Batch processing
-
-Batch processing relies on performing large, expensive computations
-infrequently. This means that you can never achieve real-time results, and it
-often does not build upon its prior computations.
-
-In contrast, Materialize continually updates queries as data comes in, which
-means it performs small, efficient computations constantly. This enables
-real-time results and builds upon all of your prior computations.
-
-### Materialized views in relational databases
-
-To maintain materialized views, most RDBMSes occasionally re-run the view's
-underlying query. This results in a potential impact to the database's
-performance while updating the view, as well as data that is only infrequently
-up-to-date.
+Materialize is a **streaming database** purpose-built for low-latency
+applications. You can use it to solve problems where data must be processed at
+speeds and scales not possible in traditional databases, but without the cost,
+complexity, and development time of most streaming engines.
+
+If you need to reduce the time it takes to refresh queries that run frequently,
+or trigger user-facing actions as soon as events happen, Materialize might be a
+good fit for you! Rather than recalculate results from scratch each time it's
+queried, or serve stale cached results, Materialize continually ingests data
+and keeps results up to date for you as **incrementally updated materialized
+views**.
+
+<br>
+
+| ![streaming_analytics.gif](https://user-images.githubusercontent.com/23521087/233637537-4c2e5133-5a93-4d9d-9800-a0d4ba842c82.gif) |
+|:---------------------------------------------------------------------------------------------------------------------------------:|
+| *As an example, you can use Materialize to keep your dashboards automatically up to date.*                                                           |
+
+## Key features
+
+Materialize combines all the desirable features of SQL databases and abstracts
+away all the hard parts required to build a streaming engine that is
+horizontally scalable, highly available and strongly consistent. To get up and
+running, all you need to know is **SQL**.
+
+### Incremental updates
+
+Materialized views are useful to avoid re-running heavy queries that run
+frequently against a database, and typically use caching to serve results
+faster. In traditional databases, materialized views often come with a long
+list of limitations and force you to make a compromise between the freshness of
+the results, the cost of refreshing the view and the complexity of the SQL
+statements you can use.
```

Comment:
```suggestion

Creating materialized views avoids re-running frequent heavy queries against a
database because they cache queries to deliver faster results. In traditional
databases, materialized views limit the complexity of SQL statements you can
use and require time and resources to return the most up-to-date results.
```